zookeeper-server/src/main/java/org/apache/zookeeper/server/DataTree.java

```diff
@@ -364,57 +368,16 @@ public void updateCount(String lastPrefix, int diff) {
             return;
         }
         synchronized (node) {
+            // todo nixon?
```

Comment:
Is this line to be removed?

Reply:
Good catch! I forgot to remove my "read this" marker.